__tests__/commands/install/integration.js

```diff
@@ -794,14 +794,14 @@ test.concurrent('install should not continue if integrity check passes', (): Pro
     let reinstall = new Install({}, config, reporter, await Lockfile.fromDirectory(config.cwd));
     await reinstall.init();
 
-    expect(await fs.exists(path.join(config.cwd, 'node_modules', 'yarn.test')));
+    expect(await fs.exists(path.join(config.cwd, 'node_modules', 'yarn.test'))).toBe(true);
```

Comment:
+1 although we use a lot of .toBe(true), we probably should do a one-off search and replace for all places like that